pkg/kubelet/server/server_test.go

```diff
@@ -132,10 +132,6 @@ func (fk *fakeKubelet) GetKubeletContainerLogs(ctx context.Context, podFullName,
 	return fk.containerLogsFunc(ctx, podFullName, containerName, logOptions, stdout, stderr)
 }
 
-func (fk *fakeKubelet) GetHostname() string {
-	return fk.hostnameFunc()
```

Comment:
is this `hostnameFunc` field still needed in `fakeKubelet`? Should we remove it?

Reply:
oh yeah, I was too hasty cleaning this up, thanks. Done.